### ISSUE TITLE
Give a warning if bindActions fails to bind?

### ISSUE THREAD

**Issue by bebraw** [goatslacker/alt#617]
Let's say you have code like this:

```javascript
class LaneStore {
  constructor() {
    this.bindActions(LaneActions);
  }
  ...
}
```

It would be useful if Alt gave warnings for missing bindings during development. Now it's easy to typo a method and miss it given it fails silently.

**Comment by goatslacker** [comment]
I've thought about that but then that means that you'll have to add every method in the actions to the store.

**Comment by bebraw** [comment]
Is there a case where that's a good practice? Can you give a concrete example?

**Comment by goatslacker** [comment]
Not all actions map 1:1 to stores.

```js
UserActions
  userAuthorized
  userUpdated

UserStore
  bindActions(UserActions)

  userUpdated(user) {
    this.setState({ user })
  }


CompanyStore
  bindActions(UserActions)

  userAuthorized() {
    this.setState({ users: this.state.users + 1 })
  }
```

There's also potentially the case where an action is handled in multiple stores and `bindActions` is lazily used, we wouldn't want to require the definition of all other actions.

**Comment by bebraw** [comment]
Yeah, great point. What if you could mark actions as strict somehow? Maybe there's a neat way to solve it on definition level. This will make it more verbose, but it will also communicate intent far better.

**Comment by goatslacker** [comment]
I'm cool with that :) `.isRequired` lol

**Comment by bebraw** [comment]
> I'm cool with that :) .isRequired lol

Yeah. I would be happy with a little schema definition. Safer to use, harder to break.
